.github/workflows/ci.yml

```diff
@@ -1,15 +1,21 @@
 name: CI
 
 on:
-  pull_request:
   push:
     branches:
       - main
   workflow_dispatch:
     inputs:
-      amdgpu_families:
+      linux_amdgpu_families:
+        type: string
+        description: "Insert comma-separated list of Linux GPU families. ex: gfx942X, gfx1201X"
+      windows_amdgpu_families:
         type: string
-        default: gfx94X-dcgpu
+        description: "Insert comma-separated list of Windows GPU families. ex: gfx942X, gfx1201X"
```

Comment:
Can you keep defaults on these? It will be easier to trigger and test if the default works.